### ISSUE TITLE
Web app: Safari: New note menu always shows a focus indicator

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#16378]
### Operating system

macOS

### Joplin version

3.7.0 (dev, web)

### Current behaviour

Currently, opening the new note menu in Safari shows an orange focus indicator, even if not navigating with the keyboard:

https://github.com/user-attachments/assets/e6ed0c51-beb2-4276-980a-517691694603

### Expected behaviour

As when running in Chrome:
- The focus indicator should not be shown when navigating with a mouse or touchscreen.
- The focus indicator should be shown when navigating with a keyboard or similar input device (e.g. screen reader).

**Comment by priyanshusky0** [comment]
Hi @personalizedrefrigerator , I’d like to work on this issue. I can investigate the Safari-specific focus behaviour and compare it with the existing Chrome behaviour, including adding regression coverage if appropriate.  
Please let me know if anyone is already working on this or if you’re okay with me taking it up.

**Comment by personalizedrefrigerator** [comment]
@priyanshusky0 -- see https://github.com/laurent22/joplin/issues/15625.

My attempt at a fix was to try passing [`focusVisible: false`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus#focusvisible) when calling `.focus` to set the default focus location. However, that breaks the focus indicator in Chrome when navigating with the keyboard.

**Comment by personalizedrefrigerator** [comment]
Upstream issues: https://bugs.webkit.org/show_bug.cgi?id=247416, https://bugs.webkit.org/show_bug.cgi?id=309245
